core-ajax.html

```diff
@@ -255,46 +304,27 @@
        * @type Array
        * @default []
        */
-      activerequests: {
+      activeRequests: {
         type: Array,
-        notify: true
+        notify: true,
+        readOnly: true,
+        value: function() {
+          this._setActiveRequests([]);
+        }
       }
     },
 
-    configure: function () {
-      return {
-        url: '',
-        params: {},
-        method: 'GET',
-        headers: {},
-        contenttype: 'application/x-www-form-urlencoded',
-        body: undefined,
-        sync: false,
-        handleas: 'json',
-        withcredentials: false,
-        auto: false,
-        verbose: false,
-        lastrequest: null,
-        lastresponse: null,
-        lasterror: null,
-        activerequests: []
-      };
-    },
-
-    bind: {
-      url: 'requestOptionsChanged',
-      method: 'requestOptionsChanged',
-      headers: 'requestOptionsChanged',
-      contenttype: 'requestOptionsChanged',
-      body: 'requestOptionsChanged',
-      sync: 'requestOptionsChanged',
-      handleas: 'requestOptionsChanged',
-      withcredentials: 'requestOptionsChanged'
+    observers: {
+      'url method headers contentType body sync handleAs withCredentials':
+        'requestOptionsChanged'
     },
 
-    ready: function () {
-      this._afterReady = true;
-      this.requestOptionsChanged();
+    configure: function() {
+      return {
+        boundHandleResponse: this.handleResponse.bind(this),
```

Comment:
Use underscore prefixes? Maybe we should decide on a common syntax for binding event handlers since it's a common thing to do.